init.rb

```diff
@@ -5,17 +5,24 @@
   name 'Redmine Issue Evm plugin'
   author 'Hajime Nakagama'
   description 'This is a plugin for Redmine. Earned value management using the ticket of redmine.'
-  version '3.4.1'
+  version '3.5'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.